plugins/search/src/components/SearchContext/SearchContextForStorybook.stories.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ApiProvider } from '@backstage/core-app-api';
+import { SearchResultSet } from '@backstage/search-common';
+import { TestApiRegistry } from '@backstage/test-utils';
+import React, { ComponentProps } from 'react';
+import { searchApiRef } from '../../apis';
+import { SearchContextProvider as RealSearchContextProvider } from './SearchContext';
+
+type QueryResultProps = {
+  mockedResults?: SearchResultSet;
+};
+
+/**
+ * Utility context provider only for use in Storybook stories.
```

Comment:
Do you think its valuable here with a link to the real SearchContextProvider? or do you think that the user, when looking at the stories, will not go here at all?

Reply:
Good idea.